api/envoy/config/core/v3/base.proto

```diff
@@ -358,6 +358,16 @@ message HeaderValueOption {
   // existing values. Otherwise it replaces any existing values.
   // This field is deprecated and please use
   // :ref:`append_action <envoy_v3_api_field_config.core.v3.HeaderValueOption.append_action>` as replacement.
+  //
+  // .. attention::
+  //   The :ref:`external authorization service <envoy_v3_api_msg_service.auth.v3.CheckResponse>` and
+  //   :ref:`external processor service <envoy_v3_api_msg_service.ext_proc.v3.ProcessingResponse>` have
+  //   error default value (``false``) for this field.
+  //   And the default value of field
+  //   :ref:`append_action <envoy_v3_api_field_config.core.v3.HeaderValueOption.append_action>`
+  //   is always ``APPEND_IF_EXISTS_OR_ADD``.
+  //   It means that the default behavior of ``HeaderValueOption`` in these two services will be changed after
+  //   this field is removed in the future version of API.
```

Comment:
Per other comments, it's not going to get removed, so let's remove this. It's still not clear to me what we are telling the end user. Are we just telling them that the default is false when used in these 2 services? If so can we just say that clearly?